validator/src/main/resources/expected-data-template/python/ecs/hc-trace.mustache

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "name": "^{{serviceName}}$",
+    "http": {
+      "request": {
+        "url": "^http://localhost:8080/$",
+        "method": "^GET$"
+      },
+      "response": {
+        "status": "^200$"
+      }
```

Comment:
Let's not validate response status, there are many reasons this can come out different and we don't generally care that the response status is different so long as the rest of the telemetry is fine.

Reply:
Updated